### PR TITLE
Fixes part of #1367: Remove the boilerplate "isTablet" if statements from the code (ProfileChooser Part)

### LINKED ISSUE #1367: Remove the boilerplate "isTablet" if statements from the code
Refactor the code which detects whether the device is a tablet or not.  
## Example
**Old code**
```
val marginMax = if (activity.resources.getBoolean(R.bool.isTablet)) {
        if (orientation == Configuration.ORIENTATION_PORTRAIT) {
          (activity as Context).resources.getDimensionPixelSize(R.dimen.home_tablet_port_outer_margin)
        } else {
          (activity as Context).resources.getDimensionPixelSize(R.dimen.home_tablet_land_outer_margin)
        }
      } else {
        if (orientation == Configuration.ORIENTATION_PORTRAIT) {
          (activity as Context).resources.getDimensionPixelSize(R.dimen.margin_28)
        } else {
          (activity as Context).resources.getDimensionPixelSize(R.dimen.margin_72)
        }
      }
```
**New code** would be a single value for the `marginMax` in the `dimens.xml` file but overridden in different configurations
## Checklist  
- [x] ProfileChooser
- [x] RecentlyPlayed
- [x] HomeScreen

### PR DESCRIPTION
**Pull request by MohamedMedhat1998**
## Explanation

- Refactored `ProfileChooserFragmentPresenter` code by removing the boilerplate `if` statements that check the type of the device and replacing them with XML values overridden in different configurations.  
- Fixes #1367 
## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/main/res/values-xhdpi/dimens.xml [hunk 1]
```diff
@@ -2,5 +2,4 @@
 <resources>
   <dimen name="enter_text_view_padding_start">120dp</dimen>
   <dimen name="enter_text_view_padding_end">120dp</dimen>
-  <integer name="profile_chooser_span_count">5</integer>
```
Comment (rt4914) [inline app/src/main/res/values-xhdpi/dimens.xml:5]: Why was this removed?

File: app/src/main/res/values/dimens.xml [hunk 2]
```diff
@@ -6,7 +6,8 @@
 
   <integer name="fade_in_duration_ms">1000</integer>
   <integer name="fade_out_duration_ms">1000</integer>
-  <integer name="profile_chooser_span_count">4</integer>
+  <integer name="profile_chooser_span_count">2</integer>
```
Comment (rt4914) [inline app/src/main/res/values/dimens.xml]: Transfer all integers to `integer.xml` file for this PR and previous PRs too.

**Review by rt4914 — CHANGES_REQUESTED** [review]
One of the major change in this PR is to shift the `integer` values to `integer.xml` file instead of `dimens.xml` file. And this applies to your other new/old PRs too.

File: app/src/main/res/values-xhdpi/dimens.xml (hunk 1 above)
Comment (MohamedMedhat1998) [reply app/src/main/res/values-xhdpi/dimens.xml:5]: It was moved to instead `values-land-xhdpi/dimens.xml` of `values-xhdpi/dimens.xml` because we only update the span count based on the density in the landscape mode only.

File: app/src/main/res/values/dimens.xml (hunk 2 above)
Comment (MohamedMedhat1998) [reply app/src/main/res/values/dimens.xml]: Ok, I'll do it now

Comment (MohamedMedhat1998) [reply app/src/main/res/values/dimens.xml]: Done

**Comment by MohamedMedhat1998** [comment]
@rt4914 PTAL  
- I have moved the integer values from `dimens.xml` to their proper files in `integers.xml`  
- I have updated the description of the PR to be `Fixes #1367` instead of `Fixes part of #1367`

File: app/src/main/res/values/dimens.xml [hunk 3]
```diff
@@ -3,10 +3,6 @@
   <!-- Default screen margins, per the Android Design guidelines. -->
   <dimen name="activity_horizontal_margin">16dp</dimen>
   <dimen name="activity_vertical_margin">16dp</dimen>
-
-  <integer name="fade_in_duration_ms">1000</integer>
-  <integer name="fade_out_duration_ms">1000</integer>
```
Comment (rt4914) [inline app/src/main/res/values/dimens.xml:8]: `fade_in` , `fade_out` you have shifted but you have not updated the implementation code where they are getting used and maybe there are not even getting used. So to be on save side, I suggest shifting it back to its original so that it does not create any issues for others.

**Review by rt4914 — APPROVED** [review]
LGTM, just one revert change.

File: app/src/main/res/values/dimens.xml (hunk 3 above)
Comment (MohamedMedhat1998) [reply app/src/main/res/values/dimens.xml:8]: Ok, I'll update it
